content/en/real_user_monitoring/guide/understanding-the-rum-event-hierarchy.md

```diff
@@ -0,0 +1,112 @@
+
+Notice that each session is automatically associated with a unique `session.id`.
+
+## Views
+Within a session, a view event is created each time a user navigates to a page (Browser RUM) or screen/screen segment (Mobile RUM) of an application. 
+
+Each view automatically collects multiple view specific attributes and data, such as text in the URL and timing metrics, like the load time of a given page. When querying for specific views, you can add any default level attributes, like device, operating system, user information, etc. However, for event-specific attributes, these must be view-specific. To switch over to view events only, you can adjust the event selector as shown in the video below.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-switch-views.mp4" alt="RUM views" video="true" style="width:80%;">}}
+
+Similarly to the `session.id`, each view will automatically have a unique `view.id` connected to it. 
+
+### Actions, errors, resources, and long tasks
+
+Within views, the SDK creates more granular events that all fall along the same level hierarchically. However, each event is unique and carries its own attributes and properties.
+
+### Actions
+
+Actions represent user activity on a page. In browsers, all click actions are automatically collected. On mobile, all taps, swipes, and scrolls are collected. Beyond these default actions, you can also send [custom actions][5], such as form completion and business transactions. 
+
+#### Sample search: Top list of all "Add to cart" actions that led to an error 
+
+In this example, search within the actions event for an action error count (of at least 1) and specific action name to see how many times clicking "Add to cart" led to a user encountering an error.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-actions-all-add-to-cart.png" alt="Diagram of the RUM event hierarchy." style="width:80%;">}}
+
+### Errors
+
+You can use RUM to collect [frontend errors][6] that occur during the user session. By default, the browser SDK creates error events for unhandled exceptions and console errors. Additionally, you can collect custom errors through the RUM addError API ([browser][7] | [mobile][8]). On mobile apps, you can also see if the error led to a session termination, also known as a crash.
+
+Errors can be viewed across both RUM and Error Tracking. Source and custom errors will be processed by Error Tracking, while console errors will be solely in RUM.
+
+#### Sample search: List of all crashes that occurred on the "CheckoutViewController"page in the iOS Shop.ist application
+
+In this example, search within the errors event for the application ID, crashes, and view (page) name to see all crashes that occurred on the "CheckoutViewController" page for a particular operating system and application.
```

Comment:
left this suggestion a few min ago on google doc but for 61, it's a big long and isn't super clear. how about this:
_In this example, search within the errors event to see all crashes that occurred on the “CheckoutViewController” page for a particular operating system  and application. _